docs/processen/Delft AVG.md

```diff
@@ -50,36 +50,44 @@ Voor het verwerken van de aanvragen wordt VJV gebruikt. De aanvraag wordt tevens
 
 ![Proces](./bestanden/Delft-Inzageverzoek/Proces%20view%20Inzageverzoek%20AVG%20v2.png)
 
-* De aanvrager dient via de website (VJV) een inzageverzoek persoonsgegevens in en geeft (na authenticatie via DigiD) op welke onderdelen geanalyseerd moeten worden. [US #153](https://github.com/VNG-Realisatie/gemma-zaken/issues/153);
+* De aanvrager (initiator) dient via de website (VJV) een inzageverzoek persoonsgegevens in en geeft (na authenticatie via DigiD) op welke onderdelen geanalyseerd moeten worden. [US #153](https://github.com/VNG-Realisatie/gemma-zaken/issues/153);
 * De VJV wordt, na correct en volledige registratie, toegewezen aan de afdeling ‘Advies JZ’ en geregistreerd in VJV, de ZRC  en de Domeinspecifieke gegevens. [US #153](https://github.com/VNG-Realisatie/gemma-zaken/issues/153), Status ‘Geregistreerd’.
 * Advies JZ routeert de VJV naar de Behandelaar van inzageverzoeken. Dit wordt geregistreerd in VJV en de ZRC. US #..., geen statuswijziging;
```

Comment:
Hier ontbreekt nog een US referentie

Reply:
Sergei schreef: 

> Hier ontbreekt nog een US referentie.

Wellicht moet ik die referentie eruit halen. Deze actie is wat ons betreft overbodig. De Behandelaar is lid van de groep 'Advies JZ'. In de praktijk opent de Behandelaar het verzoek en wijst het aan zichzelf toe.

Reply:
Prima om referentie weg te halen dan :)

Reply:
Done!